apps/public-reference/pages/applications/review/view-application.tsx

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Let's move this to something like /applications/view — I think we'll want to use it for referring to applications after time has passed, so it's not really part of the review flow.

Reply:
Sounds great, I changed the URL to /applications/view.